app/api_topologies.go

```diff
@@ -255,6 +268,30 @@ type topologyStats struct {
 	FilteredNodes      int `json:"filtered_nodes"`
 }
 
+// AddContainerFilters adds to the topology registry's containerFilters
+func AddContainerFilters(newOptions []APITopologyOption) {
+	topologyRegistry.addContainerFilters(newOptions)
+}
+
+func (r *registry) addContainerFilters(newOptions []APITopologyOption) {
+	r.Lock()
+	defer r.Unlock()
+
+	for _, o := range newOptions {
+		ContainerOpts = append(ContainerOpts, o)
+	}
+
+	var c = r.items["containers"]
+	var h = r.items["containers-by-hostname"]
+	var i = r.items["containers-by-image"]
+	c.Options[0].Options = ContainerOpts
```

Comment:
I'm not sure why it was made like this, but containerFilters is of type []APITopologyOptionGroup. However, each option group has an ID, as if it was intended to be a map instead of an array